### PR TITLE
[SD-2993] Fixing the send and sendandcontinue actions.

### PR DESCRIPTION
**Pull request by mdhaman**

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

File: client/app/scripts/superdesk-authoring/tests/authoring_spec.js [hunk 1]
```diff
@@ -922,4 +922,109 @@ describe('authoring actions', function() {
...
+            };
+
+            privileges.setUserPrivileges(userPrivileges);
+            $rootScope.$digest();
+            var itemActions = authoring.itemActions(item);
+            allowedActions(itemActions, ['new_take', 'save', 'edit', 'duplicate', 'view', 'spike',
+                    'mark_item', 'package_item', 'multi_edit', 'publish']);
+        }));
+
+    it('Cannot send item if the no move privileges',
```
Comment (amagdas) [inline client/app/scripts/superdesk-authoring/tests/authoring_spec.js]: This test name should be rephrased

**Comment by amagdas** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**